discord/ext/commands/core.py

```diff
@@ -819,6 +819,23 @@ def reset_cooldown(self, ctx):
             bucket = self._buckets.get_bucket(ctx.message)
             bucket.reset()
 
+    def get_cooldown_seconds_left(self, ctx):
+        """Retrieves the amount of seconds before this command can be tried again.
+
+        Parameters
+        -----------
+        ctx: :class:`.Context`
+            The invocation context to retrieve the cooldown from.
+
+        Returns
+        --------
+        Optional[:class:`float`]
+            The amount of time left on this command's cooldown in seconds
+        """
+        if self._buckets.valid:
+            bucket = self._buckets.get_bucket(ctx.message)
+            return bucket.get_seconds_left()
```

Comment:
```suggestion
            current = ctx.message.created_at.replace(tzinfo=datetime.timezone.utc).timestamp()
            return bucket.get_retry_after(current)
```

This made me realize that `is_on_cooldown` doesn't have this done either due to oversight. Would be good if this could be done as well there.